docs/DEVELOPMENT.md

```diff
@@ -479,4 +482,4 @@ on:
         - devel
 ```
 
-NOTE: Create a secret called ``ACCESS_TOKEN`` in repo>settings>secrets with a [token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) generated by a user with write privileges to enable the automated push to the gh-pages branch.
\ No newline at end of file
+NOTE: Create a secret called ``ACCESS_TOKEN`` in repo>settings>secrets with a [token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) generated by a user with write privileges to enable the automated push to the gh-pages branch.
```

Comment:
I don't get what changed here, GH shows texts which are identical,

Reply:
I see it now, there was new newline at the end of file. Editor sometimes adds it without user action